production/aggregator/README.md

```diff
@@ -0,0 +1,13 @@
+### Create Sealed secret
+
+```shell
+cd tmp
+
+export ENVIRONMENT_NAME=production
+
+kubectl create secret generic postgres-certs --namespace ${ENVIRONMENT_NAME} --dry-run=client --type=kubernetes.io/tls --from-file=tls.crt=postgres-production-client-cert.pem --from-file=tls.key=postgres-production-client-key.pem --from-file=ca.crt=postgres-production-ca.pem -o yaml | kubeseal --controller-name=sealed-secrets --controller-namespace=sealed-secrets --format yaml > sealed-secret-postgres-certs.yaml
```

Comment:
Substitute the text `production` with the variable `${ENVIRONMENT_NAME}` so the documentation works for both environments.